### PR TITLE
🌱 Fix comment issue in controlplane/kubeadm/internal/etcd/etcd.go

### PR DESCRIPTION
**Pull request by chymy**
**What this PR does / why we need it**:
Fix comment issue in controlplane/kubeadm/internal/etcd/etcd.go
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by sbueringer** [comment]
/lgtm
/ok-to-test

Looks like we're missing some very basic linters. (although I'm not sure there is one which only enforces godoc conventions we care about).

File: controlplane/kubeadm/internal/etcd/etcd.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ type MemberAlarm struct {
 type AlarmType int32
 
 const (
-	// AlarmOK denotes that the cluster member is OK.
+	// AlarmOk denotes that the cluster member is OK.
 	AlarmOk AlarmType = iota
```
Comment (enxebre) [inline controlplane/kubeadm/internal/etcd/etcd.go:72]: shouldn't we actually rename the variable to `AlarmOK` according to convention and [speeling](https://en.wikipedia.org/wiki/OK#:~:text=OK%20(spelling%20variations%20include%20okay,or%20a%20sign%20of%20indifference.)?

Comment (sbueringer) [reply controlplane/kubeadm/internal/etcd/etcd.go:72]: I think you're right

**Comment by fabriziopandini** [comment]
@chymy you are doing a great cleanup work!
What about grouping those fixing in bigger bulk, so we can make more discrete improvements and get the most out of reviewers time?

**Comment by sbueringer** [comment]
> @chymy you are doing a great cleanup work!
> What about grouping those fixing in bigger bulk, so we can make more discrete improvements and get the most out of reviewers time?

Potential further improvement of this could be to find out if there is a linter we could use via golangci-lint for some of these findings. Then we could introduce the linter and fix issues of the same "kind" all at once (or split them up if they are just too many and introduce the linter in the last PR)

**Comment by chymy** [comment]
Comment issues seems to be too many. Executing the `make link` command does not check for comment issues.
I use `golint` is work.

File: controlplane/kubeadm/internal/etcd/etcd.go (hunk 1 above)
Comment (chymy) [reply controlplane/kubeadm/internal/etcd/etcd.go:72]: I will fix.

**Comment by fabriziopandini** [comment]
> Comment issues seems to be too many. Executing the make link command does not check for comment issues.
I use golint is work.

@chymy if you are interested in this area please chime in the discussion around https://github.com/kubernetes-sigs/cluster-api/issues/4622, so we can agree on a set of linters checks we rely on, while everything else is discretional (including warning from other linters).

/approve

**Comment by chymy** [comment]
> > Comment issues seems to be too many. Executing the make link command does not check for comment issues.
> > I use golint is work.
> 
> @chymy if you are interested in this area please chime in the discussion around #4622, so we can agree on a set of linters checks we rely on, while everything else is discretional (including warning from other linters).
> 

OK, thanks.

**Pull request merged**